frontend/src/component/common/ConstraintAccordion/ConstraintOperator/ConstraintOperator.tsx

```diff
@@ -1,33 +1,47 @@
 import { IConstraint } from 'interfaces/strategy';
 import { formatOperatorDescription } from 'component/common/ConstraintAccordion/ConstraintOperator/formatOperatorDescription';
-import { useStyles } from 'component/common/ConstraintAccordion/ConstraintOperator/ConstraintOperator.styles';
 import React from 'react';
+import { styled } from '@mui/material';
 
 interface IConstraintOperatorProps {
     constraint: IConstraint;
     hasPrefix?: boolean;
 }
 
+const StyledContainer = styled('div')(({ theme }) => ({
+    padding: theme.spacing(0.5, 1.5),
+    borderRadius: theme.shape.borderRadius,
+    backgroundColor: theme.palette.constraintAccordion.operatorBackground,
+    lineHeight: 1.25,
```

Comment:
maybe we can use `lineHeight` from `theme.typography`. default one or our overrides. https://mui.com/material-ui/customization/default-theme/